nextjsUIapp/src/app/components/config-home.yaml

```diff
@@ -43,8 +43,10 @@ boxiconsstatisticscount:
     short_description: ""
     sparql_query: |-
       PREFIX biolink: <https://w3id.org/biolink/vocab/> 
-      select DISTINCT (COUNT(?s) as ?count) where {
-        ?s biolink:iri ?o.
+      select  (COUNT(DISTINCT(?o)) as ?count) where {
+        GRAPH <https://www.portal.brain-bican.org/grapidreleasev01> {
```

Comment:
![Screenshot 2024-12-03 at 11 39 59 AM](https://github.com/user-attachments/assets/f89e615d-bad8-4404-8d3a-b3cce5169f2d)

I got this on https://www.portal.brain-bican.org/grapidreleasev01 -- is the link correct?

Reply:
That url is for named graph ( `grapidreleasev01` is not tied to anything ) and it's fine that it is not resolvable.